DOCSP-33172 added meta description for top 250 pages in docs-mongodb-shell (#307)
mongodb/docs-mongodb-shell: source/install.txt

   :values: tutorial

.. meta:: 

>    :description: Prerequisites, compatibility issues, and how to install mongosh, the MongoDB shell.

Reviewer: ```suggestion
   :description: Prerequisites, compatibility issues, and installation instructions for mongosh, the MongoDB Shell.
```
